docs/hardware/oak-d-pro.md

```
@@ -264,6 +264,6 @@ under all reasonably foreseeable conditions. No additional PPE is required for n
 
 ## Revision History
 
-| Date | Author | Change |
+ Date | Author | Change |
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: trailing_only; Missing leading pipe

(MD055, table-pipe-style)